reference-architecture/aws-ansible/playbooks/roles/s3-registry-bucket/tasks/main.yaml

```diff
@@ -1,6 +1,10 @@
 ---
 - name: set the name of the bucket
-  set_fact: s3_bucket="{{ ansible_date_time.epoch }}-openshift-docker-registry"
+  set_fact: s3_bucket="{{ ansible_date_time.epoch }}-openshift-docker-registry-{{ stack_name}}"
```

Comment:
hrm. @detiber it feels like if I move s3 bucket creation up to the cloudformation play and output the bucket name it will allow for us to use the cloudformation lookup and not repeat the creation step. So even if we use epoch at time of creation it will always be defined in the cfn output?

Thoughts? 

cc @stuartbfox